raven_framework

```diff
@@ -11,5 +11,54 @@ SCRIPT_DIR=`(cd $SCRIPT_DIRNAME; pwd)`
 
 source $SCRIPT_DIR/scripts/setup_raven_libs
 
-python $SCRIPT_DIR/framework/Driver.py "$@"
+# set up run keywords
+# "ARGS" stores command line arguments not treated in this file (passed through)
+declare -a ARGS
+# by default run in "opt" mode
+MODE='opt'
+# loop through arguments and look for those that are handled before Python is called
+for var in "$@"; do
+  if [ "$var" = '-D' ]; then
+    # run in development mode (assertions not stripped)
+    MODE='dev'
```

Comment:
MODE is set to 'dev' but not referenced in the later code.

Reply:
I turned the "mode" check into a "case" statement and exit on a catch-all, so dev is explicitly used.